packages/bpk-component-popover/readme.md

```diff
@@ -42,23 +42,30 @@ class App extends Component {
 
   render() {
     return (
-      <BpkPopover
-        id="my-popover"
-        target={
-          <BpkButton onClick={this.openPopover}>Open</BpkButton>
-        }
-        onClose={this.closePopover}
-        isOpen={this.state.isOpen}
-        label="My popover"
-        closeButtonText="Close"
-      >
-        <BpkText>My popover content</BpkText>
-      </BpkPopover>
+      <div>
+        <BpkPopover
+          id="my-popover"
+          target={
+            <BpkButton onClick={this.openPopover}>Open</BpkButton>
+          }
+          onClose={this.closePopover}
+          isOpen={this.state.isOpen}
+          label="My popover"
+          closeButtonText="Close"
+          renderTarget={this.renderTarget}
+        >
+          <BpkText>My popover content</BpkText>
+        </BpkPopover>
+        <div ref={(target) => { this.renderTarget = target; }} />
```

Comment:
This extra `div` is extraneous surely? You can use the one defined on line 45 no?

Reply:
could do, but that would give the wrong idea of wrapping a `BpkPopover` with a div for the `renderTarget`, the outer `div` is a container, `BpkPopover` and the `renderTarget` are siblings, happy to change it tough!

Reply:
See the example https://github.com/Skyscanner/backpack/pull/347/files#diff-0ef12b7e9b89d736d7a6cb0987c78837R70